sdk/cosmos/azure-cosmos-kafka-connect/src/main/java/com/azure/cosmos/kafka/connect/implementation/CosmosAadAuthConfig.java

```diff
@@ -5,21 +5,36 @@
 
 import com.azure.cosmos.implementation.apachecommons.lang.StringUtils;
 
+import java.util.HashMap;
+import java.util.Map;
+
 import static com.azure.cosmos.implementation.guava25.base.Preconditions.checkArgument;
 
 public class CosmosAadAuthConfig implements CosmosAuthConfig {
+    private static final Map<CosmosAzureEnvironment, String> ACTIVE_DIRECTORY_ENDPOINT_MAP;
```

Comment:
Please remove the map - Customers will have to provide explicit Uri to AAD - the reason is that the Air-Gapped cloud teams conisder any Uris for AAD, ARM etc secrets for Air-Gapped clouds (even when they exist in dozens of publicly available GitHub repos). So, the source code should not contain any Uris - customers will have to explicitly set it.

Reply:
Never mind - just saw that this is just refactoring and the map also does not contain air-gapped clouds - just public clouds. So, this is acceptable.